build/components.ts

```diff
@@ -81,15 +69,15 @@ async function buildComponents() {
       const entry = path.resolve(p, './index.ts')
       if (!fs.existsSync(entry)) return
 
-      const external = (id) => {
-        return (
-          id.startsWith(VUE_REGEX) ||
-          id.startsWith(VUE_MONO) ||
-          id.startsWith(EP_PREFIX) ||
-          externals.some((i) => id.startsWith(i))
-        )
+      const rollupConfig = {
+        input: entry,
+        plugins,
+        external: getExternals({ full: false }),
```

Comment:
Why not `getExternals(false)`? Seems like there won't be any extension on this.

Reply:
Maybe we can customize the specified package later.
```
getExternals({ pkg: ['aa', 'bb', 'cc'] })
```